### PR TITLE
AB#1853 Added the third step for the 2nd part of the form

### PR DESCRIPTION
**Pull request by jeremiahbooth-insight**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

#1853 Create the application type form section 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] API unit tests
- [x] UI unit tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings or errors
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: UI/libs/core-public/tests/unit/components/form-steps/SecondFormStepOne.spec.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      data() {
+        return {
+          valid: false,
+        };
+      },
+    });
+  });
+
+  it('Should match the snapshot', () => {
+    expect(wrapper.html()).toMatchSnapshot();
```
Comment (sgaddameedi-insight) [inline UI/libs/core-public/tests/unit/components/form-steps/SecondFormStepOne.spec.ts:34]: Can u pls run the test case so it creates snapshot and commit it?

File: UI/libs/shared/utils/src/lib/lists/defaultList.ts [hunk 2]
```diff
@@ -28,9 +28,9 @@ export const formOneStepNames = [
 ];
 
 export const formTwoStepName = [
-  ' Employement & Weapons',
-  ' Files',
+  ' Employment & Weapons',
```
Comment (sgaddameedi-insight) [inline UI/libs/shared/utils/src/lib/lists/defaultList.ts:31]: can we rename `lists` to `constants`?

File: UI/package.json [hunk 3]
```diff
@@ -108,6 +109,7 @@
     "vue-cli-plugin-vuetify": "2.4.1",
     "vue-template-compiler": "^2.6.11",
     "vuetify-loader": "^1.7.2",
+    "webpack-bundle-analyzer": "^4.6.1",
```
Comment (sgaddameedi-insight) [inline UI/package.json:112]: you can remove the plugin, this is slowing the server down. We can use webpack CLI to analyze.

**Review by sgaddameedi-insight — CHANGES_REQUESTED** [review]

File: UI/libs/core-public/tests/unit/components/form-steps/SecondFormStepOne.spec.ts (hunk 1 above)
Comment (jeremiahbooth-insight) [reply UI/libs/core-public/tests/unit/components/form-steps/SecondFormStepOne.spec.ts:34]: odd let me try again

File: UI/libs/shared/utils/src/lib/lists/defaultList.ts (hunk 2 above)
Comment (jeremiahbooth-insight) [reply UI/libs/shared/utils/src/lib/lists/defaultList.ts:31]: we can

**Comment by jeremiahbooth-insight** [comment]
Ok should be good now

**Comment by jeremiahbooth-insight** [comment]
and re-rebased too

**Review by sgaddameedi-insight — APPROVED** [review]
LGTM
